src/styles/styles.js

```diff
@@ -2052,6 +2052,10 @@ const styles = {
         height: 20,
     },
 
+    communicationsLinkText: {
+        width: 320,
```

Comment:
I am strictly against using the fixed width. I would prefer to keep the app flexible (using flexbox) as much as possible.

Any special purpose for using this?